test/inductor/test_cpu_repro.py

```diff
@@ -46,6 +46,7 @@
 check_model = test_torchinductor.check_model
 
 
+@patch("torch._dynamo.config.dynamic_shapes", False)
```

Comment:
The blanket here makes me nervous, since it's easy for people to add new tests to this class and not realize they've implicitly turned off dynamic shapes.